### PR TITLE
Adding field group_welcome_message

### PR DESCRIPTION
**Pull request by anvmn**
https://issuetracker.amplexor.com/jira/browse/CFM-1249

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.features.field_base.inc [hunk 1]
```diff
@@ -82,6 +82,25 @@ function c4m_content_group_field_default_field_bases() {
     'type' => 'entityreference',
   );
 
+  // Exported field_base: 'c4m_group_welcome_message'.
+  $field_bases['c4m_group_welcome_message'] = array(
```
Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.features.field_base.inc]: @nader77 @soniCaH 
Should we move the base field to a different feature ?

**Comment by anvmn** [comment]
@nader77 
Ready for review.

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.features.field_base.inc (hunk 1 above)
Comment (soniCaH) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.features.field_base.inc]: @anvmn not if they are only used in one content type (which is the case here), so they can stay here!

Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.features.field_base.inc]: thanks

**Pull request merged**